runbotics/runbotics-orchestrator-ui/src/src-app/utils/findVariablesInActivities.ts

```diff
@@ -0,0 +1,74 @@
+import {
+    BPMNElement,
+    CamundaInputParameter,
+    CamundaOutputParameter,
+    CamundaParameter,
+    IBpmnGateway,
+    IBpmnSubProcessBusinessObject,
+    ISequenceFlowBusinessObject,
+    getParameterValue
+} from '#src-app/views/process/ProcessBuildView/Modeler/helpers/elementParameters';
+
+export const findVariablesInGateway = (businessObject: IBpmnGateway, searchPhrase: string): boolean => {
+    const gatewayOutgoingFlows: ISequenceFlowBusinessObject[] = businessObject.outgoing as unknown as ISequenceFlowBusinessObject[];
+
+    const gatewayExpressions = gatewayOutgoingFlows
+        .map(flow => flow.conditionExpression.body.toLowerCase())
+        .filter(expression => expression.includes(searchPhrase));
+
+    return gatewayExpressions.length > 0;
+};
+
+export const findVariablesInLoop = (element: BPMNElement, searchPhrase: string): boolean => {
+    const loopVariable = (element.businessObject as IBpmnSubProcessBusinessObject).loopCharacteristics.loopCardinality.body;
```

Comment:
same here, are you sure `element.businessObject` is of type IBpmnSubProcessBusinessObject? I see that it should be BusinessObject